doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/api/ContextManager.kt

```diff
@@ -7,10 +7,16 @@ import kotlin.js.JsExport
 interface ContextManager {
 
     fun setAuthToken(token: String)
+    fun isAuthTokenAboutToExpire(): Boolean
     fun setRefreshToken(token: String)
-    fun setOperationContext(userId: String, certificateChain: List<String>, privateKey: ByteArray)
-    fun setOperationContextJson(data: String)
     fun setFusionAuthToken(token: String)
+    fun isFusionAuthTokenAboutToExpire(): Boolean
```

Comment:
The fusion auth token is actually encrypted, the expiry is not visible to the end user. 

Here is a horrible example:

```
eyJhbGciOiJFQ0RILUVTK0ExMjhLVyIsImVuYyI6IkExMjhDQkMtSFMyNTYiLCJraWQiOiJlNjc4ZTBiMC1iYzYyLTRjZTktYTE3Yi1jZDNlNTY1NzBhNTQiLCJjdHkiOiJKV1QiLCJlcGsiOnsia3R5IjoiRUMiLCJ4IjoiUmZjZXd5SV9xUXlNMkF5TmVrcG1xRjRhUzRKaEtmZVl6T1pWNzJhWXU4NCIsInkiOiJ2RzNNQ0tySUpEaTNKaEpFaWk4NllaQW1RRkdnaWFjT2JoRGxpQzNxZjkwIiwiY3J2IjoiUC0yNTYifX0.J-aAu_v_nQcAX5_tCEycSfQz3gnuBIf1Pgn19WKmlRvuX8ib5Yvdcg.kD9xVPa2Pmwu_N3BE3dxRA.dXqk1XrvVOtcf-x7u2I7K6M4M2GeVWEA0NZOOH02ZAr68YTtV6Ic29kuAPaSFjfdcLRzfTyqCE_WJWWqx87MIuV5z_wPblwNRxfxC9UrxBaZKknL0QAUf_oNCqIZO5M3s6IXlOsuoM0eF-ZwRfzr0C9HADPXByv1dL0LyjPlzLAs7EtWy3LZAC5dMLrLRkSYPFYUUHP6NAzarTf5-PmGZ4ALtWAGNqGM0C3621SoH-MCKOhdaeZYVBXLjX617pF-L5J9cWbJlZakcz7rH3L1QoI32eoB86d2nsOqmaIa6jQtiMhoTuokVx2_howVa63LQZ8KBPuSP3dW9txuZW0CxK6tF4_EBX3Q_2GMB_wl2Z44QjumMuRR9vgoqEEVbCZkhh5rn_U30ahVieXiUdDwzaEBkFePLzeqIWPkZqFnJMOEkXnf8pFi5Ee9f9whuFFPjjcsIz5ivIuzWKQuDuLTTMr6oZX5EdG2gqcrfspXbfJYjL7OCn32ynJXlH8HBJozP7TpXcQQBsqFdLwv7ojWBbT69xcdtujTXtN-iWiTrIkS2lG2KFIag91RjNWVcJ08vB5i10dqYH08s9osXl5Ev_b019TyHJq5X5KtqnPrh6p7L2_a3GZb98a0eRIZS-0G6wzM18CT9Iik4JNWATbWJpAe4taEppXGuLVOXDl5MOvBJaK5F2BSE2w8QXyiUhbvvD02i44isWZrL9TvJoxLifslV30aIxqXtytMQA7sB26wtW6VsEXxxGt_CBO9Xz10xwdpSbB_j6ir5K4NwbQ4cXC3uMZfrIoSh2TNSKJ-RDDXvYXxxfFiuI0o_Rm0dNC8K7LPMYV9-X6XC97w50bFnrzhV8ialRJFzJjabAypDv9Qlg3U0m2fYaMc4ycQ9u2sGmw-js7M7bqOs2Topzgj7myf3ax8YUVSzUvRd_aIp6ebAFqO3mnVO3YTPOvdijikx1yYNqWxu5QItm4oDo2YQy0dnJCOAFrOni3dDTL4xUNcMgZErn8hAOdvd16v6QN6D3vGz0dzdjkT4ruNnxsUuWT_7kX2DXBsbtsuMBfdwTlvtugE0DW6kohmos8SvoaQVxiFcfi5eCPCmkuOy1w512p6j8gcH-ThQDokJsO-XdmMJabsZSqUCjxFtvsBklQpBbSwVBSgrAf1DQ1Ziv3qTnI_7FY2Gj5HSOGvtzPuMTsahebijevhFOmFHkHScQLRFc07JeXwymcI0nxBbOrna7mOiTvpN3aJcS3jyRGI6YCn3ZORPZopiG6VUjoVjDGT1tIFuoLdZSKrQOl9FtmqTo_a9m7gmRZRzXMyveT8LXe6qL1FDiYNUQk4-ij3Y8F7pyHG65gRJFV-ZyVnCrp2nPi3q-CdGlM7Myv5mRM0CeI7Zi4iNMcNPr_y8LcZa6DFwhFuqHCSdmx_IUB0IXyi7DRMICbzUE6f0h27UOkVtokNt-9jGDmAsuEuStTL1rk1Pt4eGo7x3DF84R4ObibxX4UD4Cz1YwFiSE7c7MSlnTKGdE8Yx6r4Yd_MXuVah4VWjWGnWZLHuSJr1t2GjKsRNu9t2rtrYA9dYjYbvffXpho3v9ibPPObjAESi_6VCRpMkgZtG04lRKcqxVd-MUcOjkjw1rqpPmbQeOGiMsSuvlomaM24ynxsaJdcwDcIXq1FqaExMDYluANSXd27cPrZLPeqHgWBlDpPMo5yLUZU2Z_pWb7nux3Wm_TfrcCwL2wofj8hxPIbamUdxGfWpweWpo_H5IL2MhLDTYdevjNdoVedholZhuEHZYI0__HunHlDU7H-oCDS2zNx0pkOogkMzJfinEs7eYJ83E2hU-vB4wHS7BehuDWZiXqJ92ytkomzGVGQBZOfRyOKYatg3qrwykzRLZK_oHcc2SMbhHcu7rXF6ai1r1aj3BC7tJJMV_hYGhMWEqxz61G0wnb4ffYK7kN8WjLBUQ-vNdeIPf2IDrAHpnoFQ7lLxDBIjtHuyECFU8FzpYnGN922Co3rYUN27drKkzsZccRLs1GPxG-WRbZyg03gAETaJzg3kgJ0MoRkeChB7-a6B-fRgt4QjXK_YgRNAoKxtMNUPGnRHaNw1Rj1P4oQyZF-lGn2HatdyB6mbLEFZknDBDNvN3xkP-gl0pGP5-RvH7-0GpClvdOMgkQ.3t3iF2bdxQ6PUAphKyWqOQ
```

Reply:
Got it, I’ll remove it.